src/casp/services/deploy/Dockerfile.pytorch

```diff
@@ -1,17 +1,22 @@
-FROM bitnami/pytorch:latest
+FROM pytorch/pytorch:2.0.0-cuda11.7-cudnn8-runtime
```

Comment:
Is this supposed to be a cpu-only pytorch docker? what's the difference between `Dockerfile.pytorch` and `Dockerfile.pytorch_cuda`?

Reply:
Yes, `Dockerfile.pytorch` is pytorch only on CPU, `Dockerfile.pytorch_cuda` is pytorch with cuda.

Reply:
... but this looks like to be cuda as well?!